docs/application/tizen-studio/release-notes/3-1-release-notes.md

```diff
@@ -0,0 +1,78 @@
+# Tizen Studio 3.1 Release Notes
+
+-   Release Date: Jan 15, 2019
```

Comment:
Please update this line with actual date

Reply:
Hello,
I talked with Mr. Chang-seok Oh. he told that document team will update the date when tizen studio is released